cla_backend/apps/call_centre/tests/admin/test_views.py

```diff
@@ -10,17 +10,26 @@ class OperatorAdminViewTestCase(TestCase):
     def setUp(self):
         super(OperatorAdminViewTestCase, self).setUp()
 
-        def make_op(username, is_manager=False, is_cla_superuser=False):
+        def make_op(username, is_manager=False, is_cla_superuser=False, **kwargs):
             return make_recipe(
                 "call_centre.operator",
                 user__username=username,
                 is_manager=is_manager,
                 is_cla_superuser=is_cla_superuser,
+                **kwargs
             )
 
         self.django_admin = make_user(is_staff=True, is_superuser=True)
+        self.foo_organisation = make_recipe("call_centre.organisation", name="Foo org")
+        self.foo_organisation.save()
```

Comment:
What does `.save()` achieve here? Is there a post-save action that we need to make sure is kicked off? If not, `make_recipe()` persists the object (whereas I believe it's `prepare_recipe()` that doesn't)

This will apply in a number of other places too, if I'm right

Reply:
I didn't know that make_recipe was persisting the object.

Amended